pick.lic

```diff
@@ -32,7 +32,11 @@ class LockPicker
       return
     end
 
-    lockpick_buffs = @settings.lockpick_buffs
+    lockpick_buffs = if @settings.waggle_sets['pick']
```

Comment:
I don't think you really need this var any more (or the arg to `do_buffs`). Since `@settings` is available you can do the check in `do_buffs`.